src/main/java/com/booksaw/betterTeams/customEvents/post/PostTeamDepositEvent.java

```diff
@@ -0,0 +1,48 @@
+package com.booksaw.betterTeams.customEvents.post;
+
+import com.booksaw.betterTeams.Team;
+import com.booksaw.betterTeams.TeamPlayer;
+import com.booksaw.betterTeams.customEvents.TeamMoneyEvent;
+import com.booksaw.betterTeams.customEvents.TeamPlayerEvent;
+import org.bukkit.event.HandlerList;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * An event which is called after a player has deposited money into their {@link Team}'s balance.
+ * Contains information about the amount deposited and the player who made the deposit.
+ * This event cannot be cancelled since it occurs after the deposit.
+ *
+ * To modify or cancel the deposit, use {@link TeamDepositEvent}.
+ *
+ * @author svaningelgem
+ */
+public final class PostTeamDepositEvent extends TeamPlayerEvent implements TeamMoneyEvent {
+    private static final HandlerList HANDLERS = new HandlerList();
+
+    private final double amount;
+
+    public PostTeamDepositEvent(final Team team, final TeamPlayer teamPlayer, final double amount) {
+        super(team, teamPlayer, false);
+
+        this.amount = amount;
+    }
+
+    public static HandlerList getHandlerList() {
+        return HANDLERS;
+    }
+
+    @Override
+    public @NotNull HandlerList getHandlers() {
+        return HANDLERS;
+    }
+
+    @Override
+    public double getAmount() {
+        return this.amount;
+    }
+
+    @Override
+    public void setAmount(final double amount) {
+        throw new UnsupportedOperationException("You can't change the amount in a post event.");
```

Comment:
I think a separate interface or something would be clearer rather than throwing an error here. 
I.e. 
`public interface TeamMoneyEvent`
and `public interface PreTeamMoneyEvent`

Reply:
fixed with a PostTeamMoneyEvent instead (because I might break the API otherwise).